UI/src/model/selectedMassTransitStop.js

```diff
@@ -149,6 +168,7 @@
       }
     });
 
+    eventbus.on('servicePointCheckbox:changed');
```

Comment:
What does this listener do?

Reply:
This should be taken out since the event is in layer file of the masstransit stop